File: marketmotormicroservices/producto-service/src/main/java/com/jma/productoservice/api/ordencompra/DetalleOrdenCompraCommandInsert.java
package com.jma.productoservice.api.ordencompra;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class DetalleOrdenCompraCommandInsert {

    private int cantidad;

    private double precioUnitario;

    private Long idProducto;

    private Long idOrdenCompra;

}

File: frontend-marketmotor/src/app/models/commands/detalleordencompra/DetalleOrdenCompraCommandInsert.ts
export class DetalleOrdenCompraCommandInsert{


    cantidad:number;
    precioUnitario:number;
    idProducto: number;
    idOrdenCompra: number;

    constructor(){
        this.cantidad = 0;
        this.precioUnitario = 0.0;
        this.idProducto = 0;
        this.idOrdenCompra = 0;
    }
    
}